Dispatcher/Controllers/Homepage/HomeViewController.swift

```diff
@@ -93,7 +127,6 @@ extension HomeViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
         return 40
```

Comment:
Better save this value as a variable in a dedicated constants file so it's easier to change / replicate on other pages